cogs/TextFilter.py

```diff
@@ -0,0 +1,701 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Text Filter Code
```

Comment:
Feel free to put your name up here ;)
```suggestion
Koala Bot Text Filter Code
Created By: Stefan Cooper
```